### ISSUE TITLE
Generate test alerts to a specific target only?

### ISSUE THREAD

**Issue by luckman212** [netdata/netdata#8667]
**TL;DR— Is it possible to test alarm notifications to just 1 specific type/target?**

I read [Netdata alarm notifications](https://docs.netdata.cloud/health/notifications/) but am stuck. Hoping someone can help figure this out. I configured 2 "notification types" in my `health_alarm_notify.conf` — MSTeams and Twilio/SMS. Both are working fine.

My question is about testing. When I run this command:
```
su -s /bin/bash netdata /usr/libexec/netdata/plugins.d/alarm-notify.sh test
```
...alerts are successfully sent to both Teams and Twilio. **What I would like to do is be able to test _JUST_ Teams or _JUST_ Twilio.** I tried adding a custom "role" at the end of my config, something like this:
```
# custom role for Teams
role_recipients_msteam[teams]="${DEFAULT_RECIPIENT_MSTEAM}"
```
and then trying to send an alert to _just_ that role using:
```
su -s /bin/bash netdata /usr/libexec/netdata/plugins.d/alarm-notify.sh test "teams"
```
But alerts are still sent to ALL types. Am I misunderstanding how this is supposed to be set up, or is there any way to achieve this? Thank you.

**Comment by thiagoftsm** [comment]
Hi @luckman212 ,

Unfortunately this is not possible yet, but we are considering to change this script in a near future.

Best regards!

**Comment by luckman212** [comment]
@thiagoftsm Ok, thanks - I wasn't sure if I was just doing something wrong.
